Cargo.toml

```diff
@@ -21,8 +21,8 @@ backon = { version = "1.5.1", features = ["tokio-sleep"] }
 borsh = { version = "1.5.1", features = ["derive"] }
 clap = { version = "4.5.20", features = ["derive", "env"] }
 curve25519-dalek = "4.1.3"
-# TODO(#451): Use our own fork or upstream library
-dcap-qvl = { git = "https://github.com/mattlockyer/dcap-qvl", rev = "3caedd8fd0f365841abaf1430ff2922fcb09d9db" }
+# TODO(#815): Use upstream library
+dcap-qvl = { git = "https://github.com/gilcu3/dcap-qvl", rev = "287a6e7b00f3a687255e90733e954c979d1df9d5", default-features = false, features=["contract", "std", "report"] }
```

Comment:
Why did we fork this into your personal account and not the company org?

Reply:
We agreed on that just because it was an ephemeral fork